usaspending_api/etl/es_recipient_profile_template.json

```diff
@@ -0,0 +1,71 @@
+{
+  "settings": {
+    "index.mapping.ignore_malformed": true,
+    "index.max_result_window": null,
```

Comment:
What's the intention behind setting this value to `null`? I see this is the case on other indexes, but I'm not sure `null` is a valid value for this field. 

https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html

Reply:
I kept this as `null` since it's what we have in the other index template files. We define an actual value for our indices in **settings.py** as `<index>_MAX_RESULT_WINDOW` and set them to 50,000. My only assumption is that we don't want to define this value in multiple places?